vcd/resource_vcd_vapp_static_routing.go

```diff
@@ -0,0 +1,211 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"log"
+)
+
+func resourceVcdVappNetworkStaticRouting() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVappNetworkStaticRoutingCreate,
+		Delete: resourceVAppNetworkStaticRoutingDelete,
+		Read:   resourceVappNetworkStaticRoutingRead,
+		Update: resourceVappNetworkStaticRoutingUpdate,
+		Importer: &schema.ResourceImporter{
+			State: vappNetworkStaticRoutingImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Required: false,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"vdc": {
+				Type:        schema.TypeString,
+				Required:    false,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The name of VDC to use, optional if defined at provider level",
+			},
+			"vapp_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "vApp identifier",
+			},
+			"network_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "vApp network identifier",
+			},
+			"enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Enable or disable static Routing.",
+			},
+			"rule": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
What happens if we define this resource without any rule?

Reply:
you then can toggle `enabled`